samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,130 @@
+.. _at_client_sample:
+
+nRF9160: AT Client Sample
+#########################
+
+The AT Client sample demonstrates the asynchronous serial communication taking place over UART to the nRF9160 modem. AT client is a stand-alone proxy firmware which enables an external computer or MCU to get access to the nRF9160 modem API. 
+
+
+Overview
+********
+
+The AT Client sample uses AT commands for sending directives to the nRF9160 modem. This facilitates the reading of responses or analyzing of events related to the nRF9160 modem. The commands can be initiated from a terminal or the `LTE Link Monitor`_ which is an application implemented as part of the `nRF Connect for Desktop`_ . 
+
+For more information on the AT commands, see `AT Commands Reference`_ .
+
+Also see :ref:`modem_info_readme` for more information regarding the process of interaction between an LTE application and the modem using the modem information library.
+
+
+Requirements
+************
+
+* The following development kit board:
```

Comment:
Is this because it's standard practice in NCS not to use 'kit'? Further down its named 'nRF9160 DK board'

Reply:
It's because we have that introduction (without "kit") in the sample template and use the same sentence in all the other samples.
In general, we don't have any convention on saying "development board" or "development kit board" or "development kit" - but maybe we should have?